superset/assets/src/explore/components/ExploreViewContainer.jsx

```diff
@@ -17,6 +18,16 @@ import * as saveModalActions from '../actions/saveModalActions';
 import * as chartActions from '../../chart/chartAction';
 import { fetchDatasourceMetadata } from '../../dashboard/actions/datasources';
 import { Logger, ActionLog, EXPLORE_EVENT_NAMES, LOG_ACTIONS_MOUNT_EXPLORER } from '../../logger';
+import { match } from '../../utils/common';
+
+// Prolly need to move this to a global context
+const keymap = {
+  EXPLORE: {
+    RUN: ['command+enter'],
+    SAVE: ['command+shift'],
```

Comment:
`command-s`?

Reply:
From a user standpoint i think shift is easier, but we can go to s and i'll figure out a way to override the setttings from `superset_config.py` :)

Reply:
I would vote for `command-s`, seems more intutive.